clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -61,6 +61,8 @@ static constexpr llvm::StringLiteral InitESIMDMethodName = "__init_esimd";
 static constexpr llvm::StringLiteral InitSpecConstantsBuffer =
     "__init_specialization_constants_buffer";
 static constexpr llvm::StringLiteral FinalizeMethodName = "__finalize";
+static constexpr llvm::StringLiteral GlibcxxFailedAssertion =
```

Comment:
```suggestion
static constexpr llvm::StringLiteral LibstdcxxFailedAssertion =
```
This is a libstdc++ thing, not a glibc thing.